### PR TITLE
feat (GT-129): add generated staticmodbus.json

### PR DESCRIPTION
**Pull request by gordon-epc**
Add a generated staticmodbus.json file.

### CODE REVIEW

**Review by altendky — CHANGES_REQUESTED** [review]
Presumably the extra fields noted in https://github.com/epcpower/grid-tied/pull/59#pullrequestreview-676376372 will be dealt with here.

File: src/epcpm/sunspectostaticmodbus.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+                        ref_child, sunspec_types, staticmodbus_types
+                    )
+                    ref_child["uuid"] = generate_uuid()
+            child["uuid"] = generate_uuid()
+    table["uuid"] = generate_uuid()
+
+
+@click.command()
+@click.option("--input_sunspec_filename")
+@click.option("--output_staticmodbus_filename")
```
[the remark this answers is not in the document]
Comment (gordon-epc) [reply src/epcpm/sunspectostaticmodbus.py]: Made this change in c229af5275782dd9871f1bc720612b4a3ec84b12

File: src/epcpm/sunspectostaticmodbus.py [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+def sunspec_to_staticmodbus_generator(
+    input_sunspec_filename, output_staticmodbus_filename
+):
+    # The purpose of this script is to take as input the sunspec JSON file and
+    # output a comparable static modbus JSON file.
+    # It is most likely one time use code.
```
[the remark this answers is not in the document]
Comment (gordon-epc) [reply src/epcpm/sunspectostaticmodbus.py]: The comment added as a docstring and the contents of the string are much briefer in c229af5275782dd9871f1bc720612b4a3ec84b12

File: src/epcpm/sunspectostaticmodbus.py [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+                    print(f"ERROR: unexpected root child: {root_child['_type']}")
+
+    with open(
+        output_staticmodbus_filename, "w", encoding="utf-8"
+    ) as output_staticmodbus_fp:
+        json.dump(data, output_staticmodbus_fp, ensure_ascii=False, indent=4)
+
+
+if __name__ == "__main__":
+    sunspec_to_staticmodbus_generator()
```
[the remark this answers is not in the document]
Comment (gordon-epc) [reply src/epcpm/sunspectostaticmodbus.py]: Removed the `__main__` code. Moved the file to the `epcpm/cli` directory to match with other cli code. Added the group "utility" and put the sunspec to staticmodbus script plus a "transition" command into the "utility" group. All this done in c229af5275782dd9871f1bc720612b4a3ec84b12

**Comment by gordon-epc** [comment]
Removal of the fields mentioned in [epcpower/gridtied #59](https://github.com/epcpower/grid-tied/pull/59#pullrequestreview-676376372) has been done.

**Review by altendky — APPROVED** [review]

**Pull request merged**